Makefile

```diff
@@ -273,24 +273,29 @@ test-nightly-server: build-nightly-server
 
 ### Check Rules: static checks, quality tools ###
 
-check: check-shell-scripts
+check: check-shell-scripts check-yaml
 .PHONY: check
-
 # rule requires shellcheck and find to run
 check-shell-scripts: $(filter $(ALT_BIN)%,$(SHELLCHECK))
 	$(SHELLCHECK) -P tests/ -eSC2181 -fgcc $$(find $(ROOT_DIR) -name "*.sh")
 .PHONY: check-shell-scripts
 
+
+check-yaml: $(filter $(ALT_BIN)%,$(YAMLLINT_CMD))
+	$(YAMLLINT_CMD) -c $(CURDIR)/.yamllint.yaml $(CURDIR)
+.PHONY: check-yaml
+
 # not included in check to not disrupt wip branches
 check-gitlint: $(filter $(ALT_BIN)%,$(GITLINT))
 	$(GITLINT) -C .gitlint --commits origin/master.. lint
 .PHONY: check-gitlint
 
 ### Misc. Rules ###
 
-clean:
+clean: clean-buildfiles clean-altbin
+clean-buildfiles:
 	$(RM) $(BUILDFILE_PREFIX)*
-.PHONY: clean
+.PHONY: clean clean-buildfiles
```

Comment:
Can we bring all clean targets together at one place?

Reply:
@anoopcs9  wrote:

> good idea, I will update 



> Can we bring all clean targets together at one place?
>

good idea, I will update

Reply:
added a corresponding  commit